src/main/java/com/rit/group2/models/Employee.java

```diff
@@ -110,12 +110,8 @@ public void terminate(){
 		this.active = false;
 	}
 	
-	public Set<BasicEmployee> getWorkers(){
-		Set<BasicEmployee> basicWorkers = new HashSet<BasicEmployee>();
-		for(Employee worker: workers){
-			basicWorkers.add(new BasicEmployee(worker));
-		}
-		return basicWorkers;
+	public Set<Employee> getWorkers(){
+		return workers;
```

Comment:
Just remembered we need to add a bossId to employees in order to get the hierarchy

Reply:
@secretspy97